src/components/header-system/README.md

```diff
@@ -1,14 +1,384 @@
-# Header System
-> Intro text/summary
+# Header
 
-## Header system modules
-> Most stuff in this RFC is still applicable. Will make for a good start on usage documentation
+Add an IU-branded header and navigation to your website or application.
 
-## States and roles
-> E.g. active states, open, closed, ARIA roles, etc.
+## Overview
 
-## JavaScript
-> What's going on with the JS here?
+Branding guidelines require each site to have an IU-branded header. For more information on web branding guidelines, visit [brand.iu.edu](brand.iu.edu).
 
-## Usability guidance
-> Do this, Don't do this..
\ No newline at end of file
+## Usage
+
+A header is made up of four parts: the **base header**, **primary navigation**, **secondary navigation**, and **search**.
+
+> Only the base header is required. Navigation and search are optional.
+
+### Base header
+
+The base header contains the IU trident logo and the title of your website or application.
+
+```html
+<header class="rvt-header-wrapper">
+    
+    <!-- Hidden link for screen reader users to skip to main content -->
+    
+    <a class="rvt-header-wrapper__skip-link" href="#main-content">Skip to main content</a>
+    
+    <div class="rvt-header-global">
+        <div class="rvt-container-xl">
```

Comment:
I think we should have at least a small section/heading that covers the use of the generic `rvt-container-X` utility to constrain the max width of a site/app header.

In iu.edu we used the `rvt-container-xl` size, because of the number of navigation items, etc. The `-xl` container gave us the space we needed, but the point of using the container-x utility in the header is to allow developers/designers to set a max-width to their header to match the rest of the `rvt-container-x` elements they are using to constrain the max-width of their sites.

For example if I was building a web app that had a giant table that listed a bunch of record with lots of columns, I would probably want to use an `rvt-container-xl` to constrain the max width of the main content area (where the table lives) so as to give desktop users a lot space for the table to spread out on large monitors. On the other hand if I had a smaller site with some editorial content I might want to constrain the max width of my content to a `rvt-container-lg` element. This way I can just update the container element in the header of my site to match.

Not sure if that is making sense the way I'm describing it?

Reply:
@levimcg It does, thanks!